index.js

```diff
@@ -79,23 +83,48 @@ module.exports = (function () {
    * @param {Object} bVersions - bible version object
    * @param {String} start - start index
    * @param {String} end - end index
+   * @param {Boolean} includeSuperscript - Indicate whether verse superscript
+   *    should be returned with text.
    * @return {String} bible passage
    */
-  bible._getVerses = function (bVersions, start, end) {
+  bible._getVerses = function (bVersions, start, end, includeSuperscript) {
     var s = VERSE_INDEX[start];
     var e = VERSE_INDEX[end];
     var arr = [];
 
     // Return single verse if no end
-    if (!e) { return bVersions[s]; }
+    if (!e) { return (includeSuperscript ? this._getVerseByVerseIndex(s) : '') + bVersions[s]; }
 
-    for (var i = Number(s), len = Number(e);
-           i < len + 1; i++) {
-      arr.push(bVersions[i]);
+    var i = Number(s), len = Number(e);
+    if (includeSuperscript) {
+      for (; i < len + 1; i++) {
+        arr.push(this._getVerseByVerseIndex(i) + bVersions[i]);
+      }
+    } else {
+      for (; i < len + 1; i++) {
+        arr.push(bVersions[i]);
+      }
     }
 
     return arr.join(' ');
   };
 
+  /**
+   * Convert a verse index value into the superscript verse number associated with the passage.
+   *
+   * @param {number} value - Value of any key pair within the VERSE_INDEX object
+   * @returns {string} - The superscript verse associated with the passed index value.
+   * @private
+   */
+  bible._getVerseByVerseIndex = function (value) {
+    for (var key in VERSE_INDEX) {
```

Comment:
I don't particularly enjoy the fact that I must use a `for in` loop here, but due to the fact that VERSE_INDEX is an object that is not necessarily in incremental order I could think of no more efficient solution.